src/components/_popovers.scss

```
@@ -10,7 +10,8 @@
   &:hover {
     &::after {
       opacity: 1;
-      transition: opacity 0.2s ease-out;
+      transition: opacity 0.2s ease-out, visibility 0s linear 0s;
```

`opacity 235ms ease-in-out` are the original transition values, probably better to stick to it.
The last `0s` is not needed as it is the default. 
```suggestion
      transition: opacity 235ms ease-in-out, visibility 0s linear;
```